### PR TITLE
Add tutorial page to docs

### PR DESCRIPTION
**Pull request by hexatester**
+ Tutorials on Creating Simple plugin and Running bot on android
+ Fix Contribute link

#908 #987 
Please audit this PR, because i'm not native English speaker. Thanks.

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+var bot = mineflayer.createBot({
+  username: "Steve",
+});
+bot.loadPlugin(myNamePlugin);
+
+bot.once("spawn", function () {
+  bot.myNameIs(); // My name is Steve!
+  bot.myNameIs("Alex"); // My name is Alex!
+});
```
Comment (wvffle) [inline docs/tutorial.md:28]: Use `standard` code style

File: docs/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Tutorial
+
+Example can be found in the [examples](https://github.com/PrismarineJS/mineflayer/tree/master/examples) folder.
```
Comment (wvffle) [inline docs/tutorial.md:3]: Exmple**s**

**Comment by wvffle** [comment]
A tutorial should contain a step by step guide how to create a simple bot. Every step should be explained briefly.

**Comment by hexatester** [comment]
Ah i see.

Here is my idea on the tutorial part
1. Introduction
(short explanation about the main ways to interact with API)
2. Setup
(install & update)
3. Your first bot
(explain creation, event & callback, thought process)

Any suggestions?

File: docs/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+## Creating Simple plugin
+
+Here basic example on how to create plugin.
```
Comment (rom1504) [inline docs/tutorial.md:7]: we already have doc there https://github.com/PrismarineJS/mineflayer/blob/master/docs/CONTRIBUTING.md on how to create a plugin

**Comment by rom1504** [comment]
I think what you created here is basically the same as what we already have.
I listed there https://github.com/PrismarineJS/mineflayer/issues/908#issuecomment-628255428 what would be good to do as tutorials.
Basically what is important is to be able to clear up things that people are often confused about, enable people to do more, and reduce the amount of questions we get.

**Comment by hexatester** [comment]
Okay. I'm closing this PR.
I'll do more research on the tutorial.

**Pull request closed without merging**